cli/tsc/dts/lib.dom.d.ts

```diff
@@ -3156,6 +3156,7 @@ interface Body {
     arrayBuffer(): Promise<ArrayBuffer>;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Request/blob) */
     blob(): Promise<Blob>;
+    bytes(): Promise<Uint8Array>;
```

Comment:
This doesn't yet have an MDN reference, but when it does, it'll be at https://developer.mozilla.org/en-US/docs/Web/API/Request/bytes. Is it worth updating now or once it exists?

Reply:
Seems worth adding it now since it will eventually work.